Mathlib/Topology/Algebra/Monoid.lean

```diff
@@ -399,6 +399,103 @@ instance Submonoid.continuousMul [TopologicalSpace M] [Monoid M] [ContinuousMul
     (S : Submonoid M) : ContinuousMul S :=
   S.toSubsemigroup.continuousMul
 
+section MulZeroClass
+
+open Filter
+
+variable {α β : Type*}
+variable [TopologicalSpace M] [MulZeroClass M] [ContinuousMul M]
+
+/-- Let `M` be a topological space with a continuous multiplication operation and a `0`.
+Let `l` be a filter on `M` which is disjoint from the cocompact filter. Then, the multiplication map
+`M × M → M` tends to zero on the filter product `𝓝 0 ×ˢ l`. -/
+theorem tendsto_mul_nhds_zero_prod_of_disjoint_cocompact {l : Filter M}
+    (hl : Disjoint l (cocompact M)) :
+    Tendsto (fun (x : M × M) ↦ x.1 * x.2) (𝓝 0 ×ˢ l) (𝓝 0) := calc
+  (𝓝 0 ×ˢ l).map fun (x : M × M) ↦ x.1 * x.2
```

Comment:
We tend to prefer `map f l` over `l.map f` for consistency with the `f '' s` notation, where `f` comes first.